runtime/oti/j9nonbuilder.h

```diff
@@ -3585,6 +3585,28 @@ typedef struct J9Method {
 	U_8* bytecodes;
 	struct J9ConstantPool* constantPool;
 	void* methodRunAddress;
+
+	/**
+	 * This field is overloaded and can have several meanings:
+	 * 
+	 * 1. If the `J9_STARTPC_NOT_TRANSLATED` bit is set, this method is interpreted
+	 * 		a. If the value is non-negative, then the field contains the invocation count which is decremented towards
+	 * 		zero. When the invocation count reaches zero, the method will be queued for JIT compilation. The invocation
+	 * 		count is represented as:
+	 * 
+	 * 		```
+	 * 		(invocationCount << 1) | J9_STARTPC_NOT_TRANSLATED
+	 * 		```
+	 * 
+	 * 		b. If the value is negative, then it can be one of:
+	 * 			- `J9_JIT_NEVER_TRANSLATE`: ???
+	 * 			- `J9_JIT_QUEUED_FOR_COMPILATION`: The method has been queued for JIT compilation
+	 * 			- `-1`: ???
```

Comment:
We see this happening here:
https://github.com/eclipse/openj9/blob/250b180e1fa101e3dd3e171a3bb686b9301eab08/runtime/vm/bindnatv.cpp#L311-L320

@gacholio I see your name in the history of this file so assuming you are the expert in the area. Could you provide a short summary of what these "INL" methods are and why we set `extra` to -1? I'd like to create a `J9_JIT_<something>` define for this `-1` value so it is not a magic constant in code.

Reply:
Before any change, we need to see if the value is ever read and compared to -1 (which would also need to be changed to use the new constant). I suspect an easier change (which still requires the same analysis) would be to use the "never translate" constant here instead of -1.

Reply:
To answer the question, INL means "internal native library" which are natives that are implemented directly in the interpreter rather than via a JNI callout.

Reply:
I don't see anyone comparing to -1 directly. Using a new define for this value seems appropriate. While we're at it, it might be time for some renaming of these constants (though that may involve DDR work so maybe should be done separately). The `STARTPC` name is mixed up between two different fields, so making them explicitly `EXTRA` or `CONSTANTPOOL` constants would be nice.

Reply:
Fixed in [force-push](https://github.com/eclipse/openj9/compare/9837dbd2dc060a8d9102441fffcc446c13902523..8ea8c5df617546e5fb6907f41e50112adad4e8e6).